Dockerfile

```diff
@@ -8,7 +8,7 @@ RUN yarn install
 COPY web ./
 RUN yarn build
 
-FROM golang:1.17-alpine as backend
+FROM golang:1.22-alpine as backend
```

Comment:
Optional: The Golang version stated here (1.22) doesn't match the version in `go.mod` (1.21). Please consider setting them to the same versions. Also, consider setting the GO_VERSION as an argument within the Dockerfile so that we can override the version when building the image, preferably based on the version in go.mod for consistency reasons.